### PR TITLE
Add documentation to the Stax filters feature

### PR DESCRIPTION
**Pull request by brandelune**
This PR corresponds to the https://github.com/omegat-org/omegat/issues/421 documentation issue.

### CODE REVIEW

**Review by miurahr — CHANGES_REQUESTED** [review]
You describe the commit is updating document, but it modified Bundle.properties.
Please don't change that is not related to the purpose.

**Comment by brandelune** [comment]
I think we're good now.

**Review by miurahr — APPROVED** [review]

File: src/org/omegat/Bundle.properties [hunk 1]
```diff
@@ -1507,10 +1507,10 @@ SDLPROJECT_FILTER_NAME=SDL project
 SDLXLIFF_FILTER_NAME=SDL XLIFF
 
 # org.omegat.filters4.xml.xliff.Xliff1Filter.java
-XLIFF1FILTER_FILTER_NAME=XLIFF 1.x Bilingual
+XLIFF1FILTER_FILTER_NAME=XLIFF
```
Comment (Kazephil) [inline src/org/omegat/Bundle.properties:1511]: I thought you wanted to name them:
- XLIFF 1.x
- XLIFF 2.x

It's probably fine with XLIFF 2 right below, but just XLIFF might be misinterpreted as applying to all versions of the format.

Comment (Kazephil) [reply src/org/omegat/Bundle.properties:1511]: OK. In that case this is probably ready to be merged if you're happy with the wording tweaks I made in `App_filter.xml`.

**Comment by Kazephil** [comment]
Merged into `master`.

**Pull request closed without merging**